src/metrics_server/connection_metrics.ts

```diff
@@ -124,12 +112,15 @@ export function isValidConnectionMetricsReport(
       return false;
     }
 
-    // Check that `countries` are strings.
-    if (userReport.countries) {
-      for (const country of userReport.countries) {
-        if (typeof country !== 'string') {
-          return false;
-        }
+    // We require at least 1 country to be set
```

Comment:
As per our discussion, revert this part of the change.

Reply:
Addressing in #1532